src/heads-up/views/app-link.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <div class="app-link">
+    <panel-section title="Rel">
+      <details
+        v-for="(section, key) in appLinkData"
+        :key="key"
+        class="app-link__section"
+        open
+      >
+        <summary><ChevronRightIcon />{{ key }}</summary>
+        <ul
+          style="margin-left: 1rem; list-style: none;"
+          class="app-link__list"
+        >
+          <li
+            v-for="(item, index) in section"
+            :key="index"
+            class="app-link__list-item"
+          >
+            <span
+              v-if="item.length === 1"
+              class="app-link__attribute-value"
+            >
+              <span class="app-link__attribute">{{ item[0][0] }}</span> <span
+                class="app-link__value"
+                :title="item[0][1]"
+              >{{ item[0][1] }}</span>
+            </span>
+            <details
+              v-else
+              class="app-link__property-table"
+            >
+              <summary>
+                <ChevronRightIcon />
+                <span class="app-link__attribute-value">
+                  <span class="app-link__attribute">{{ item[0][0] }}</span> <span
+                    class="app-link__value"
+                    :title="item[0][1]"
+                  >{{ item[0][1] }}</span>
+                </span>
+              </summary>
+
+              <dl class="app-link__table-row app-link__attribute-value">
+                <template v-for="(row, rowIndex) in item">
+                  <template v-for="(column, i) in row">
+                    <!-- eslint-disable-next-line vue/no-v-html, vue/max-attributes-per-line -->
```

Comment:
I don't think this is necessary anymore. It doesn't seem to throw a linting error when I run it